### PR TITLE
bug 1519705. bump openshift-elasticsearch-plugin-2.4.4.20 and build i…

### PR DESCRIPTION
**Pull request by jcantrill**
…f necessary

(cherry picked from commit b2af1d20ec47ff9e9cd0eff846697d5214b5ca67)

release-3.6 backport of #835

### CODE REVIEW

File: elasticsearch/prep-install.origin [hunk 1]
```diff
@@ -7,6 +7,18 @@ if [ -z "${ES_CLOUD_K8S_URL:-}" ] ; then
     ES_CLOUD_K8S_URL=io.fabric8/elasticsearch-cloud-kubernetes/${ES_CLOUD_K8S_VER}
 fi
 if [ -z "${OSE_ES_URL:-}" ] ; then
-    OSE_ES_URL=io.fabric8.elasticsearch/openshift-elasticsearch-plugin/${OSE_ES_VER}
+    MANPATH="" source /opt/rh/rh-maven33/enable
+    set +e
+    mvn dependency:get -Dartifact=io.fabric8.elasticsearch:openshift-elasticsearch-plugin:${OSE_ES_VER}
```
Comment (richm) [inline elasticsearch/prep-install.origin]: This spews a lot of output . . . not sure where we can put it.  It doesn't look good in the CI output.

File: elasticsearch/prep-install.origin [hunk 2]
```diff
@@ -7,6 +7,18 @@ if [ -z "${ES_CLOUD_K8S_URL:-}" ] ; then
...
+    res=$?
+    set -e
+    if [ 0 -eq $res ]; then
+      OSE_ES_URL=io.fabric8.elasticsearch/openshift-elasticsearch-plugin/${OSE_ES_VER}
+    else
+      pushd /tmp/lib/openshift-elasticsearch-plugin
+        mvn clean verify -DskipTests
```
Comment (richm) [inline elasticsearch/prep-install.origin]: This also spews a lot of output

File: elasticsearch/prep-install.origin (hunk 1 above)
Comment (jcantrill) [reply elasticsearch/prep-install.origin]: maven doesnt seem to have a 'search' feature.  I am really not fond of this approach but its the only way i have found to determine if it is published.

File: elasticsearch/prep-install.origin (hunk 2 above)
Comment (jcantrill) [reply elasticsearch/prep-install.origin]: i believe this replicates a lot of what 'get' does.

File: elasticsearch/prep-install.origin (hunk 1 above)
Comment (richm) [reply elasticsearch/prep-install.origin]: understood - it is distressing to look at the CI log from this run and see hundreds of red lines of `[ERROR]` . . .

**Pull request merged**